content/en/core/guides/update-dependencies.md

```diff
@@ -19,7 +19,7 @@ Every minor release we update dependencies to get the latest fixes and improveme
 
 ## Steps
 
-1. `git checkout master && git pull` - get the latest code
+1. `git checkout main && git pull` - get the latest code
```

Comment:
This is for cht-core. We could make it ambiguous, eg:

```suggestion
1. Checkout and pull the latest main branch
```

Reply:
Sounds good. Maybe then to say "Checkout and pull the latest default branch"? (no reference to either master or main)

Reply:
Yeah that's perfect. I'm sure software engineers can write their own git command!